doc/source/programs/gdal_translate.rst

```diff
@@ -413,6 +413,7 @@ source band, a mask band, an alpha band will not be used during resampling
 (when using :option:`-outsize` or :option:`-tr`).
 
 .. include:: nodata_handling_gdaladdo_gdal_translate.rst
+   :start-line: 6
```

Comment:
Using this is quite fragile... Couldn't we just add a developer comment in nodata_handling_xxx.rst to provide it's title

Reply:
Yes it looks fragile but it is all the file but the title.
> Couldn't we just add a developer comment in nodata_handling_xxx.rst to provide it's title

Sorry I don't understand what you mean.

Reply:
> Sorry I don't understand what you mean.

My below suggestion of starting lines with the title with ".. " which is a developer comment, if we want to add a title, only for people who read the .rst file (which we probably don't need). Yes I believe marking as orphan should hopefully be enough

Reply:
Ah OK! That is a "writer comment" from my perspective 😄

Reply:
```suggestion
```